wg-reliability/README.md

```diff
@@ -0,0 +1,17 @@
+# wg-reliability
+
+wg-reliability has tools used to aid in transparency of stability metrics.
+
+[sippy](https://github.com/openshift/sippy) summarizes multiple test-grid dashboards with data slicing of related jobs, job runs,
+and tests.  Visit (placeholder for eventual URL) to see the kube instance. 
```

Comment:
We should probably request public IP and DNS now (given we already have support for running it).

I'm wondering if we want "sippy.k8s.io" or something more generic?